pkg/actions/lua/lakefs/catalogexport/table_extractor.lua

```diff
@@ -40,7 +40,7 @@ end
 local function get_table_descriptor(client, repo_id, commit_id, logical_path)
     local code, content = client.get_object(repo_id, commit_id, logical_path)
     if code ~= 200 then
-        error("could not fetch data file: HTTP " .. tostring(code))
+        error("could not fetch data file: HTTP " .. tostring(code) ", path: " .. logical_path)
```

Comment:
if `logical_path` is `nil` then the code will panic `attempt to concatenate a nil value`. 

```suggestion
        error("could not fetch data file: HTTP " .. tostring(code) ", path: ",  logical_path)
```